src/schema/publisher.js

```diff
@@ -29,6 +29,17 @@ const schema = new Schema({
       message: 'Invalid domain name: {VALUE}',
     },
   },
+  storyPath: {
+    type: String,
+    trim: true,
+    validate: {
+      validator(v) {
+        return `${v}`.includes('{{story.id}}') || `${v}`.includes('{{ story.id }}');
```

Comment:
This should return `true` if `v` is empty (null or undefined) otherwise this will fail on empty values. Also, this should use one regex validator, so values such as `{{ story.id}}` can be allowed

Reply:
Fine with making it use regex, but it _should_ fail on empty values.